ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -35,6 +37,93 @@ def extract_citekey(results_url):
         citekey = short_doi_url.replace('https://doi.org', 'doi:10')
     return citekey
 
+
+def check_none(value):
+    """Raises ValueError if value is type None, else returns value"""
+    if isinstance(value, type(None)):
+        raise ValueError
+    return value
+
+
+def find_country(country):
+    """ .get retrieves data as class Country
+    .search_fuzzy matching returns a list
+    try a few ways to identify a match and
+    return as soon as find something valid
+    Input is country name (string)
+    If no match found, return None"""
+    try:
+        hit = pycountry.countries.get(name=country)
+        hit = check_none(hit)
+        return hit
+    except (LookupError, ValueError):
+        try:
+            hit = pycountry.countries.get(official_name=country)
+            hit = check_none(hit)
+            return hit
+        except (LookupError, ValueError):
+            try:
+                hit = pycountry.countries.search_fuzzy(country)
+                hit = check_none(hit)
+                if type(hit) == list and len(hit) == 1:
+                    return hit[0]
+                raise ValueError
+            except (LookupError, ValueError):
+                try:
+                    hit = pycountry.countries.search_fuzzy(country + ",")
+                    hit = check_none(hit)
+                    if isinstance(hit, list) and len(hit) == 1:
+                        return hit[0]
+                    else:
+                        raise ValueError
+                except (LookupError, ValueError):
+                    return None
+
+
+def assign_ISO(countries):
+    """ Match country names with ISO codes
+    Input: series of country names
+    Returns: dictionary of matches
+    :type countries: pd.Series """
+    # Need to hard code a few countries that aren't registered using standard names, so
+    # initializing the country_codes database with these irregular values
+    country_codes = {"South Korea": "KOR", "Democratic Republic of Congo": "COD",
+                     "Democratic Republic of the Congo": "COD", "UAE": "ARE"}
+
+    # Identify the most likely 3-letter ISO code for each country
+    failed_matches = list()
+    for country in countries:
+        if country not in country_codes.keys():
+            # Need to query the pycountry package but it can fail for a
+            # few reasons. Use function to avoid LookupError issues and
+            # try all the different ways that might help to match a
+            # country name to its ISO code
+            hit = find_country(country)
+            if not isinstance(hit, type(None)):
+                country_codes[country] = hit.alpha_3
+            else:
+                failed_matches.append(country)
+    # Print warning about failures and return successes as dictionary
+    print("Could not assign country codes to:", ", ".join(failed_matches))
```

Comment:
```suggestion
    print("Could not assign country codes to: ", ", ".join(failed_matches))
```
Add whitespace here?